src/styles/blocks/header.scss

```diff
@@ -0,0 +1,14 @@
+@use '../_variables/variables' as *;
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1200px;
```

Comment:
Consider using a variable for the max-width value instead of hardcoding '1200px'. This will make it easier to maintain and update the styles in the future. Check the variables defined in your '_variables/variables.scss' file.